authorization-dotnet/EncryptingAndDecryptingStringInCSharp/Tests/StringEncryptionTests.cs

```diff
@@ -0,0 +1,20 @@
+using EncryptingAndDecryptingStringInCSharp;
+
+namespace Tests
+{
+    public class StringEncryptionTests
+    {
+        [Fact]
+        public async Task WhenDecryptingAnEncryptedText_ThenOriginalTextIsObtained()
+        {
+            var encryptionService = new StringEncryptionService();
+            const string passphrase = "myPassword";
+            const string clearText = "We use encryption to obscure a piece of information.";
+
+            var encrypted = await encryptionService.EncryptAsync(clearText, passphrase);
+            var decrypted = await encryptionService.DecryptAsync(encrypted, passphrase);
+
+            Assert.Equal(clearText, decrypted);
+        }
+    }
+}
```

Comment:
Can we please add one test for a negative scenario? For example, when using the invalid passphrase for decryption, the new test method could be called something like `WhenDecryptingAnEncryptedTextWithInvalidPassPhrase_ThenOriginalTextIsWrong`.

Reply:
fixed